subt/marsupial.py

```diff
@@ -0,0 +1,52 @@
+"""
+  Dual robots - typically a UGV taking care of UAV
+  https://en.wikipedia.org/wiki/Marsupial
```

Comment:
But this node is supposed to be used with the UAV, right? It detaches itself. Let's add this key information here (it is nowhere else, not even in the subt documentation and also we didn't get it in the issue where we asked about it).